libs/test/MobileSyncTest/src/com/salesforce/androidsdk/mobilesync/manager/SyncManagerTest.java

```diff
@@ -702,140 +702,6 @@ public void testCleanResyncGhostsForSOSLTarget() throws Exception {
         deleteRecordsByIdOnServer(new HashSet<String>(Arrays.asList(accountIds[0])), Constants.ACCOUNT);
     }
 
-    /**
```

Comment:
Moved refresh sync down target tests to their own class in the target sub package so that they can use the package level constructor which has a count ids per soql parameter.